code/modules/transport/pods/secondary_system.dm

```diff
@@ -325,6 +328,56 @@
 			break
 	..()
 
+/obj/item/shipcomponent/secondary_system/cargo/jumpseat
+	name = "Living Hold"
+	desc = "Allows the ship to load people as cargo. It is designed to integrate with the Life-Support System so it is probably safe?."
```

Comment:
Name feels a little weird, can't put my finger on it
also you have a `?.` at the end of the desc